pkg/cli/server/config_reloader.go

```diff
@@ -73,13 +76,26 @@ func (hr *HotReloader) Start() {
 
 						newConfig := config.New()
 
-						err := LoadConfiguration(newConfig, hr.filePath)
+						err := LoadConfiguration(newConfig, hr.configPath)
 						if err != nil {
 							log.Error().Err(err).Msg("failed to reload config, retry writing it.")
 
 							continue
 						}
 
+						if hr.ctlr.Config.HTTP.Auth != nil && hr.ctlr.Config.HTTP.Auth.LDAP != nil {
+							err = hr.watcher.Remove(hr.ctlr.Config.HTTP.Auth.LDAP.CredentialsFile)
```

Comment:
why always remove credential files?
this is needed only if it changed in the meantime, (hr.ctlr.Config.HTTP.Auth.LDAP.CredentialsFile != newConfig.HTTP.Auth.LDAP.CredentialsFile) right?

Reply:
I think that's right